code/game/objects/items/robot/items/storage.dm

```diff
@@ -57,18 +57,20 @@
 
 /obj/item/borg/apparatus/pre_attack(atom/atom, mob/living/user, params)
 	if(!stored)
-		var/itemcheck = FALSE
-		for(var/storable_type in storable)
-			if(istype(atom, storable_type))
-				itemcheck = TRUE
-				break
-		if(itemcheck)
-			var/obj/item/item = atom
-			item.forceMove(src)
-			stored = item
-			RegisterSignal(stored, COMSIG_ATOM_UPDATED_ICON, PROC_REF(on_stored_updated_icon))
-			update_appearance()
-			return TRUE
+		// Borgs should not be grabbing their own modules
+		if(!istype(atom.loc, /mob/living/silicon/robot))
```

Comment:
Can't believe time green merged this before I could pester you to add an early return 😔

Reply:
Conspiratie!